runtime/sema/type_test.go

```diff
@@ -690,7 +690,7 @@ func TestIdentifierCacheUpdate(t *testing.T) {
 					cachedID := semaType.ID()
 
 					// clear cached identifiers for one level
-					semaType.cachedIdentifiers = nil
```

Comment:
It looks like the test not only resets the cached identifier of the types defined in the program, but  of all types, including the built-in ones (which might be used concurrently somewhere else).

We should maybe improve the test and not reset the cached identifiers of the built-in types (`Location == nil`)

Reply:
Improved this in [ad58840](https://github.com/onflow/cadence/pull/2624/commits/ad588404e7480000f8452be2ac2989919c6e69c9)